src/content/self-hosted/guides/azure/index.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Use Azure as your Authentication Provider
+description: Guide on how use Azure as the authentication provider of your Okteto Self-hosted instance
+id: identity-provider-azure
+---
+
+import Image from '@theme/Image';
+import TiersList from '@theme/TiersList';
+
+#  Use Azure as your Identity Provider<TiersList tiers="Self-Hosted" />
+
+This guide will walk you through the steps necessary to configure Azure as the authentication provider for your instace of Okteto.
+
+Please refer to [Azure's official documentation](https://learn.microsoft.com/en-us/power-apps/maker/portals/configure/configure-openid-settings) on Open ID Connect for more information.
+
+## Prerequisites
+
+- An instance of [Okteto](https://okteto.com/docs/self-hosted/).
+- [Helm](https://helm.sh/docs/intro/install/) 3.0+ installed in your local machine.
+- Access to an [Azure account](https://portal.azure.com) with permissions to registrate applications in Azure Active Directory.
+
+## Create an App Registration
+
+Create a new app integration, using the values below:
+
+<p align="center"><Image src="/docs/self-hosted/guides/azure/images/new-app.png" alt="Create the new App Registration" /></p>
```

Comment:
don't use `demo.okteto.dev` for the URL since that uses public override, that is not fully documented (we should write a guide on that). Use `okteto.example.com`, that's what we used for the Okta guide.